src/Slices/Settings/UI/Tabs/Environment/Tab.tsx

```diff
@@ -1,39 +1,60 @@
 import React, { useContext } from "react";
-import { EnvironmentDetails, FlatEnvironment, ProjectModel, RemoteData } from "@/Core";
+import { FlatEnvironment, ProjectModel } from "@/Core";
+import { useGetEnvironmentDetails } from "@/Data/Managers/V2/Environment/GetEnvironmentDetails";
+import { useGetProjects } from "@/Data/Managers/V2/Project/GetProjects";
 import { DependencyContext } from "@/UI";
-import { RemoteDataView } from "@/UI/Components";
+import { ErrorView, LoadingView } from "@/UI/Components";
 import { EnvironmentSettings } from "./EnvironmentSettings";
 
+/**
+ * Environment tab for the Settings page
+ *
+ * It handles different states of the environment settings and project data (loading, error, success)
+ * and renders the appropriate UI for each state.
+ *
+ * @returns {React.FC} The Environment tab
+ */
 export const Tab: React.FC = () => {
-  const { queryResolver, environmentHandler } = useContext(DependencyContext);
-  const [data] = queryResolver.useOneTime<"GetProjects">({
-    kind: "GetProjects",
-    environmentDetails: false,
-  });
+  const { environmentHandler } = useContext(DependencyContext);
+  const projectsData = useGetProjects().useOneTime();
 
   const id = environmentHandler.useId();
-  const [envData] = queryResolver.useOneTime<"GetEnvironmentDetails">({
-    kind: "GetEnvironmentDetails",
-    details: true,
-    id,
-  });
-
-  return (
-    <RemoteDataView
-      data={RemoteData.merge(envData, data)}
-      label="EditEnvironment"
-      SuccessView={([environmentDetails, projects]) => (
-        <EnvironmentSettings
-          aria-label="Environment-Success"
-          environment={addProjectName(environmentDetails, projects)}
-          projects={projects}
-        />
-      )}
-    />
-  );
+
+  const envDetailsData = useGetEnvironmentDetails().useOneTime(id);
+
+  if (envDetailsData.isError) {
+    return (
+      <ErrorView
+        message={envDetailsData.error.message}
+        retry={envDetailsData.refetch}
+        ariaLabel="EnvironmentDetails-Error"
+      />
+    );
+  }
+  if (projectsData.isError) {
+    return (
+      <ErrorView
+        message={projectsData.error.message}
+        retry={projectsData.refetch}
+        ariaLabel="EnvironmentDetails-Error"
```

Comment:
same aria-label as the other one?

Reply:
from the user perspective it's the same view, the same component, so I thought there is no need in changing that. the same approach is done in Service Instance detail view, home view and few other